sorc/fvcom_tools.fd/fvcom_readme.md

```diff
@@ -19,16 +19,23 @@
  Installation depends on the netCDF library and cmake.
```

Comment:
We are placing all the 'readme' files in the ./docs directory. So rename this file as 'user_guide.md' and replace the version in ./docs.